src/components/SignIn.jsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withRouter } from 'react-router-dom';
-
 import { auth } from '../firebase/firebase';
 import Button from './interface/Button';
 import DialogInput from './interface/dialog/DialogInput';
+import Background from '../background.jpg';
+import Typography from '@material-ui/core/Typography';
```

Comment:
let's segment these imports. Typically we have 3 blocks of imports

1. React specific (React, PropTypes, etc.)
New Line
2. MaterialUI imports
New Line
3. Components we wrote / own..